guidelines_for_grant_submissions.md

```diff
@@ -31,4 +31,15 @@ In contrast, proposals which are unlikely to be accepted include:
 * Development of hardware or software which will not be released under an open licence
 * Applications which lack justification and explanation of the amount requested
 
+For grants that include the funding of hardware purchases to support Open Source software development:
+
+* The grant application should include a description of what the project aims to achieve.  For example:
+    - Support for this hardware is committed to the upstream project repository.
+    - A sustainable community of developers is formed to continue development and support of the hardware.
+    - An individual takes an active role in the development of Open Source support for the hardware, which would have otherwise been impossible due to the cost of the hardware.
+
+* An individual or group may receive at most one grant of this nature per grant cycle.
```

Comment:
The nit-picker in me says, "Well, I'll get all 10 project members to apply for one super-duper RISC-666 laptop." I presume, though, the normal process would address people trying to game the system like this.

Reply:
Yes, the normal processes would kick in to prevent this sort of thing happening, in addition to the "group" stipulation.